packages/cli-plugin-deploy-pulumi/commands/index.js

```diff
@@ -27,6 +27,10 @@ module.exports = [
                         describe: `Environment`,
                         type: "string"
                     });
+                    yargs.option("variant", {
+                        describe: `Variant (only for staged rollouts)`,
+                        type: "string"
+                    });
                     yargs.option("build", {
                         default: true,
                         describe: `Build packages before deploying`,
```

Comment:
Can we double check this? We don't see the `variant` being added to the `watch` command.

Reply:
There is a major roadblock here - pulumi automation API does not support the `watch` command at all.
So the issue regards not just staged rollouts, but the whole usage of pulumi automation API.
It will be rather difficult to replicate that - we could add a watcher using `chokidar` and redeploy on every change. With some debouncing probably.

Reply:
Oh, ok, didn't know that. It's ok @kedrzu, I'll take that one over in the upcoming weeks and see what options do we have. Chokidar is ok, yes, but the problem is that Pulumi then has to restart the program, reevaluate the sate and see what needs to be dpeloyed, which takes time. So it won't be instant like it currently is.

But as I said, don't worry about it. We'll come up with something.